linode/manager
# M3-6008: Search Marketplace apps by category, alternate name and alternate description keywords

## Description 📝
Adds the ability to search apps in Linode Marketplace by

- category,
- alternative names (ex. TF2 and team fortress 2), and
- description keywords


## Preview 📷

![image](https://user-images.githubusercontent.com/119517080/207439202-d0221035-4a2e-48e1-b0ea-eb876191e675.png)


## How to test 🧪

- Navigate to marketplace -> `/linodes/create?type=One-Click`
- Search app by category || name || alt names || alt description keywords
- Validate results
- Example: search by category (Databases) -> expected to see Database related apps

**How do I run relevant unit or e2e tests?**
  yarn cy:run
